main-support.ts

```diff
@@ -323,8 +333,8 @@ export function takeTenClips(
     concat += '[' + (current - 1) + ':v]' + '[' + (current - 1) + ':a]';
     current++;
   }
-  concat += 'concat=n=' + (totalCount - 1) + ':v=1:a=1';
-  args.push('-filter_complex', concat, saveLocation + '/' + fileHash + '.mp4');
+  concat += 'concat=n=' + (totalCount - 1) + ':v=1:a=1[v][a];[v]scale=' + screenshotHeight + ':-2[v2]';
```

Comment:
I just realised, this is now backwards (generates 300px wide, not high)!

Should be `-2:screenshotHeight`!

Reply:
I'll also confirm later whether we need -2 or -1!

Reply:
I copied this change from your other branch without looking into what it does -- feel free to edit anything in `takeTenClips` method as you see fit 👍